kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -90,13 +90,19 @@ private[kyuubi] class EngineRef(
 
   private[kyuubi] def getEngineRefId(): String = engineRefId
 
-  // Launcher of the engine
-  private[kyuubi] val appUser: String = shareLevel match {
+  // Used to look up the engine
+  private[kyuubi] val sessionUser: String = shareLevel match {
```

Comment:
`sessionUser` seems to be misleading (my fault to suggest this name) as it is not same to `kyuubi.session.user `.
We need a better name.

Reply:
How about `engineSpaceUser` or `namespaceUser`? As we use it to lookup the engine by default.